bin/replicate-postgresql.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+function try_find_file {
+  set +e
+  out="$(aws --profile govuk-integration s3 ls "s3://${bucket}/postgresql-backend/" | grep "${1}_production.gz" | sed 's/.* //' | sort | tail -n1)"
```

Comment:
How does `aws --profile integration` interact with the [new local set up for accessing AWS](https://docs.publishing.service.gov.uk/manual/gds-cli.html)? Or is this separate? I guess I should try out running the scripts. :wink:

Reply:
It probably won't interact - these scripts probably need to be `gds-cli`'d at some point

Reply:
Yeah - oops - my GDS CLI tutorial told people to delete their `~/.aws/` config files. 😬I'll test these out and do a separate PR for any updates based on `gds-cli` stuff. This is good to merge as-is though, I don't want this to block anything as I doubt everyone would have _actually_ deleted stuff, and this is Important. Also I don't know when I'll get to testing these out as it feels Large.